docs/Reference/Rest.md

```diff
@@ -22,6 +22,215 @@ The default location is `http://localhost:9000/swagger-ui`.
 
 You can also use tools such as [Postman] or [cURL] to interact with Web3Signer APIs.
 
+## Endpoint examples
+
+### `/api/v1/eth2/publicKeys`
+
+Returns a hex-encoded list of ETH2 BLS public keys for the private keys that have been loaded into Web3Signer.
+
+!!! example
+
+    === "curl request"
+
+        ```bash
+        curl -X GET http://localhost:9000/api/v1/eth2/publicKeys
+        ```
+
+    === "JSON result"
+
+        ```json
+        [
+        "string"
+        ]
+        ```
+
+### `/api/v1/eth2/sign/{identifier}`
+
+Signs data for the ETH2 BLS public key specified as part of the URL and returns the signature.
+
+#### Path Parameter
+
+`identifier`: *string* - Key for which data to sign
+
+#### Body Parameters
+
+The request body has a `type` which defines the remaining parameters in the body.
+The following example uses `"type": "AGGREGATION_SLOT"`.
+Refer to the [API documentation](https://consensys.github.io/web3signer/web3signer-eth2.html#operation/ETH2_SIGN) for more information on the types.
+
+!!! example
+
+    === "curl request"
+
+        ```bash
+        curl -X POST http://127.0.0.1:9000/api/v1/eth2/sign/{identifier} --header "Content-Type: application/json"
+        --data '{
+          "type": "AGGREGATION_SLOT",
+          "fork_info": {
+            "fork": {
+              "previous_version": "string",
+              "current_version": "string",
+              "epoch": "string"
+            },
+            "genesis_validators_root": "string"
+          },
+          "signingRoot": "string",
+          "aggregation_slot": {
+            "slot": "string"
+          }
+        }'
+        ```
+
+    === "JSON result"
+
+        ```json
+        {
+          "signature": "0xb3baa751d0a9132cfe93e4e3d5ff9075111100e3789dca219ade5a24d27e19d16b3353149da1833e9b691bb38634e8dc04469be7032132906c927d7e1a49b414730612877bc6b2810c8f202daf793d1ab0d6b5cb21d52f9e52e883859887a5d9"
+        }
+        ```
+
+### `/eth/v1/keystores` methods
+
```

Comment:
Think we should mention that these methods require enabling using the --key-manager-api-enabled command line when using eth2 mode.